Algorithm.CSharp/OptionRenameRegressionAlgorithm.cs

```diff
@@ -112,18 +107,5 @@ public override void OnData(Slice slice)
                 }
             }
         }
-
-        /// <summary>
-        /// Order fill event handler. On an order fill update the resulting information is passed to this method.
-        /// </summary>
-        /// <param name="orderEvent">Order event details containing details of the evemts</param>
-        /// <remarks>This method can be called asynchronously and so should only be used by seasoned C# experts. Ensure you use proper locks on thread-unsafe objects</remarks>
-        public override void OnOrderEvent(OrderEvent orderEvent)
```

Comment:
Any particular reason why we're removing this logging? I'm not strongly opinionated one way or another, just curious as to the reasoning.

Reply:
I remove it, because there are plenty of those in other examples and this algorithm is a regression one.

Reply:
I think it's nice to have the order event logging in regression algorithms. Consider the regression test fails, the first thing I would want to do is compare the trades and security additions/removals to get a sense of *why* it failed.